<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYHoldingDockWidget</class>
 <widget class="QDockWidget" name="QPANSOPYHoldingDockWidget">
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>260</height>
   </size>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing"><number>4</number></property>
    <property name="margin"><number>4</number></property>
    
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title"><string>Input</string></property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0"><widget class="QLabel" name="routingLabel"><property name="text"><string>Routing:</string></property></widget></item>
       <item row="0" column="1"><widget class="QgsMapLayerComboBox" name="routingLayerComboBox"/></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="paramsGroup">
      <property name="title"><string>Parameters</string></property>
      <layout class="QFormLayout" name="paramsLayout">
       <item row="0" column="0"><widget class="QLabel" name="iasLabel"><property name="text"><string>IAS (kt):</string></property></widget></item>
       <item row="0" column="1"><widget class="QLineEdit" name="iasLineEdit"><property name="text"><string>195</string></property></widget></item>
       
       <item row="1" column="0"><widget class="QLabel" name="altLabel"><property name="text"><string>Altitude:</string></property></widget></item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="altLayout">
         <item><widget class="QLineEdit" name="altitudeLineEdit"><property name="text"><string>10000</string></property></widget></item>
         <item><widget class="QComboBox" name="altitudeUnitCombo"><item><property name="text"><string>ft</string></property></item><item><property name="text"><string>m</string></property></item></widget></item>
        </layout>
       </item>

       <item row="2" column="0"><widget class="QLabel" name="isaLabel"><property name="text"><string>ISA Var (°C):</string></property></widget></item>
       <item row="2" column="1"><widget class="QLineEdit" name="isaVarLineEdit"><property name="text"><string>0</string></property></widget></item>

       <item row="3" column="0"><widget class="QLabel" name="bankLabel"><property name="text"><string>Bank Angle (°):</string></property></widget></item>
       <item row="3" column="1"><widget class="QLineEdit" name="bankAngleLineEdit"><property name="text"><string>25</string></property></widget></item>

       <item row="4" column="0"><widget class="QLabel" name="timeLabel"><property name="text"><string>Leg Time (min):</string></property></widget></item>
       <item row="4" column="1"><widget class="QLineEdit" name="legTimeLineEdit"><property name="text"><string>1.0</string></property></widget></item>

       <item row="5" column="0"><widget class="QLabel" name="turnLabel"><property name="text"><string>Turn:</string></property></widget></item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="turnLayout">
         <item><widget class="QRadioButton" name="rightTurnRadio"><property name="text"><string>Right</string></property><property name="checked"><bool>true</bool></property></widget></item>
         <item><widget class="QRadioButton" name="leftTurnRadio"><property name="text"><string>Left</string></property></widget></item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="outputGroup">
      <property name="title"><string>Output</string></property>
      <layout class="QHBoxLayout" name="outLayout">
       <item><widget class="QLineEdit" name="outputFolderLineEdit"/></item>
       <item><widget class="QPushButton" name="browseButton"><property name="text"><string>...</string></property></widget></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="actionsGroup">
      <property name="title"><string>Actions</string></property>
      <layout class="QVBoxLayout" name="actionsLayout">
       <item><widget class="QPushButton" name="calculateButton"><property name="text"><string>Calculate</string></property></widget></item>
    <item><widget class="QPushButton" name="copyWordButton"><property name="text"><string>Copy to Word</string></property></widget></item>
      </layout>
     </widget>
    </item>

    <item>
     <widget class="QGroupBox" name="logGroup">
      <property name="title"><string>Log</string></property>
      <layout class="QVBoxLayout" name="logLayout">
       <item><widget class="QTextEdit" name="logTextEdit"><property name="readOnly"><bool>true</bool></property><property name="minimumHeight"><number>100</number></property></widget></item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 </ui>